scripts/smoke-test-webgl.py

```diff
@@ -37,6 +38,7 @@
 class RequestVerifier:
     __requests = []
     __testNumber = 0
+    __lock = Lock()
```

Comment:
Why the lock?

Reply:
it's using a multithreaded server (see end of script) and there was a data race causing the collected numbers to be off